frontend/packages/knative-plugin/src/utils/create-eventsources-utils.ts

```diff
@@ -23,7 +27,7 @@ export const getEventSourcesDepResource = (formData: EventSourceFormData): K8sRe
     sink: { knativeService },
   } = formData;
 
-  const defaultLabel = getAppLabels(name, applicationName);
+  const defaultLabel = getAppLabels(name, applicationName, '');
```

Comment:
@nemesis09 why `''` in third argument? it'll add below label am not sure if we need that

```
'app.kubernetes.io/name': '',
'app.openshift.io/runtime': '',
```

Reply:
updated the `getAppLabels` function instead of passing empty string